src/main/java/duke/DeleteCommand.java

```diff
@@ -0,0 +1,28 @@
+package duke;
+
```

Comment:
Perhaps you can include a Javadoc comment briefly describing this class?

Reply:
Alright thanks